api/lib/infrastructure/serializers/xml/xml-organization-learner-set.js

```diff
@@ -38,7 +41,12 @@ class XMLOrganizationLearnersSet {
   _check(xmlNode) {
     const nationalStudentId = _getValueFromParsedElement(xmlNode.ID_NATIONAL);
     const sexCode = _getValueFromParsedElement(xmlNode.CODE_SEXE);
+    const birthCountryCode = _getValueFromParsedElement(xmlNode.CODE_PAYS);
+    const birthCityCode = _getValueFromParsedElement(xmlNode.CODE_COMMUNE_INSEE_NAISS);
 
+    if (birthCountryCode === FRANCE_COUNTRY_CODE && isEmpty(birthCityCode)) {
```

Comment:
Pour etre plus explicite
```suggestion
    if (frenchBornHasEmptyCityCode({birthCountryCode, birthCityCode)) {
```